### DISCUSSION TITLE
Confused about Clones (and views)

### DISCUSSION THREAD

**Discussion by The5ftDev** [community/community#75232] [New to GitHub] [answered]
Hi there! I'm new to Github, and recently made my first repository, and was excited to see if it would get much attention. I found the insights button, which helped me out a little, and told me I had 2 unique views. At first I thought this was fine, 2 people is better than 0, then I found the clones graph.

It said I had 11 unique clones, which is a little higher then 2 lol. So I left it for a bit, confused, came back to it, and the issue was still there. I'm not really worried about it, I'm just curious on what's happening.

I found only one post about the traffic tab, and it was unclear how views are counted.

Thanks for any help!

**Comment by Hsin-Hung** [comment, marked as answer]
**Views** are the total number of times your repo has been viewed, which includes multiple views by the same user. 

**Unique Visitors/Views** only count each user once, regardless of how many times they visit the repo.

**Unique clones** can be higher than unique visitors because you can clone the repo as long as have access to the cloning URL without even visiting the repo itself. For example, some automated bots or scripts regularly clone public repos through GitHub API endpoints to monitor public events on GitHub for search indexes or archives. Also, direct sharing of cloning URLs will contribute to a higher number of unique clones without increasing unique visitors.

Hope it helps!

**Reply by VidathChamikara** [reply]
If anyone has not github account and view the repo via URL , then how this one count ?

**Comment by SaifurRR** [comment]
you can check this to have an idea of your path forward:
https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository

**Comment by sreyouh** [comment]
My repo have around  2000  unique visitors and 3000+ visitors and 950+ unique cloners is that natural ratio?
